src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3166,6 +3167,31 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <!--
     ============================================================
+                                        _GenerateCompilerDependencyCache
```

Comment:
I'd name this `GenerateCompileDependencyCache` (no `r`), because it's around the `Compile` target. But I could go a different way.